pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix

```diff
@@ -2077,12 +2099,12 @@
   };
   sql = buildGrammar {
     language = "sql";
-    version = "0.0.0+rev=caf2938";
+    version = "0.0.0+rev=25be0b8";
```

Comment:
this update introduces a segfault for me in sql treesitter 

```
 Storage: /var/lib/systemd/coredump/core.nvim.1001.61068abe6c0d4c6aa15c31767747dc7b.71136.1699427435000000.zst (present)
  Size on Disk: 1.5M
       Message: Process 71136 (nvim) of user 1001 dumped core.
                
                Module libstdc++.so.6 without build-id.
                Module sql.so without build-id.
                Module libgcc_s.so.1 without build-id.
                Module libluajit-5.1.so.2 without build-id.
                Module libunibilium.so.4 without build-id.
                Module libtree-sitter.so.0 without build-id.
                Module libmsgpack-c.so.2 without build-id.
                Module libvterm.so.0 without build-id.
                Module libtermkey.so.1 without build-id.
                Module libluv.so.1 without build-id.

```
i'll try to follow up with upstream to see whats happened

Reply:
https://github.com/DerekStride/tree-sitter-sql/issues/221

Reply:
segfault has been fixed, and tree-sitter-sql has published a new release. This will be picked up next time tree-sitter grammars get updated, no urgency on my side

Reply:
I have an ongoing PR to update vim plugins and treesitter grammars: #267226.